src/assets/data/markdown/ionic-weather-step08-01.md

```diff
@@ -30,29 +26,6 @@ import { StatusBar } from '@ionic-native/status-bar';
 import { SplashScreen } from '@ionic-native/splash-screen';
 import { IconMapProvider } from '../providers/icon-map/icon-map';
 import { WeatherProvider } from '../providers/weather/weather';
```

Comment:
Did we remove the error monitoring at some prior lab? I thought we had that in there still.

FWIW, I am more than happy to remove it, but just want to make sure it is not something that is being added earlier on in the process.

Reply:
Basically LGTM otherwise.